versions/1/manifest.spec.yml

```diff
@@ -233,4 +233,5 @@ spec:
   - title
   - description
   - version
-  - release
\ No newline at end of file
+  - release
+  - owner
```

Comment:
Shouldn't this be `owner.github` as this is the only property we currently use? https://github.com/elastic/package-storage/blob/production/packages/apache/0.1.4/manifest.yml#L49

Reply:
+1 

same question here

Reply:
Yup, good catch! Will fix, thanks!

Reply:
Fixed in 57f1dcc.